clients/py/tests/test_pdutils.py

```diff
@@ -29,16 +29,20 @@ def gen_dfs():
 
 def test_concat_dfs():
     dfs = list(gen_dfs())
-    df = pdutils.concat_dfs(dfs)
+    backends = ['tsdb', 'csv', 'kv', 'http', 'grpc']
```

Comment:
http and grpc are not valid backends. and stream is a backend
(some for all other places)